x-pack/plugins/apm/public/components/app/TransactionDetails/WaterfallWithSummmary/TransactionTabs.tsx

```diff
@@ -135,6 +135,7 @@ function LogsTabContent({ transaction }: { transaction: Transaction }) {
       startTimestamp={startTimestamp - framePaddingMs}
       endTimestamp={endTimestamp + framePaddingMs}
       query={`trace.id:"${transaction.trace.id}" OR "${transaction.trace.id}"`}
+      height={640}
```

Comment:
Any reason for that specific size? Is it relative to other containers or does that number just work well?

Reply:
400 is too slim for space we typically have in this panel, and it's divisible by 8 😊 Do you think it's too tall?

Reply:
No it's fine. Just curious. Feel free to merge when green.